packages/0xcert-ethereum-asset-ledger/src/tests/core/ledger/revoke-abilities-instance-method.test.ts

```diff
@@ -36,26 +38,52 @@ spec.before(async (stage) => {
 
 spec.before(async (stage) => {
   const accounts = await stage.web3.eth.getAccounts();
+  stage.set('owner', accounts[0]);
   stage.set('bob', accounts[1]);
+  stage.set('jane', accounts[2]);
 });
 
 spec.test('revokes ledger abilities for an account', async (ctx) => {
   const ledger = ctx.get('ledger');
   const bob = ctx.get('bob');
-  await ledger.grantAbilities(bob, [AssetLedgerAbility.CREATE_ASSET, AssetLedgerAbility.UPDATE_URI_BASE]).then(() => ctx.sleep(200));
-  await ledger.revokeAbilities(bob, [AssetLedgerAbility.UPDATE_URI_BASE]).then(() => ctx.sleep(200));
+  await ledger.grantAbilities(bob, [GeneralAssetLedgerAbility.CREATE_ASSET, GeneralAssetLedgerAbility.UPDATE_URI_BASE]).then(() => ctx.sleep(200));
+  await ledger.revokeAbilities(bob, [GeneralAssetLedgerAbility.UPDATE_URI_BASE]).then(() => ctx.sleep(200));
   const abilities = await ledger.getAbilities(bob);
-  ctx.deepEqual(abilities, [AssetLedgerAbility.CREATE_ASSET]);
+  ctx.deepEqual(abilities, [GeneralAssetLedgerAbility.CREATE_ASSET]);
 });
 
 spec.test('revokes ledger abilities for an order gateway', async (ctx) => {
   const ledger = ctx.get('ledger');
   const gateway = ctx.get('gateway');
   const proxyId = ctx.get('protocol').xcertCreateProxy.instance.options.address;
-  await ledger.grantAbilities(gateway, [AssetLedgerAbility.CREATE_ASSET]);
-  await ledger.revokeAbilities(gateway, [AssetLedgerAbility.CREATE_ASSET]).then(() => ctx.sleep(200));
+  await ledger.grantAbilities(gateway, [GeneralAssetLedgerAbility.CREATE_ASSET]);
+  await ledger.revokeAbilities(gateway, [GeneralAssetLedgerAbility.CREATE_ASSET]).then(() => ctx.sleep(200));
   const abilities = await ledger.getAbilities(proxyId);
   ctx.deepEqual(abilities, []);
 });
 
+spec.test('revokes ledger super abilities for an account', async (ctx) => {
```

Comment:
Please add `fails revoking ledger super ability` test.

Reply:
With current setup you cannot fail..

Reply:
Ow ... true ...